go.mod

```
@@ -81,9 +81,9 @@ require (
 	github.com/hashicorp/vault-plugin-auth-kerberos v0.1.6
 	github.com/hashicorp/vault-plugin-auth-kubernetes v0.7.1-0.20200921171209-a8c355e565cb
 	github.com/hashicorp/vault-plugin-auth-oci v0.5.5
-	github.com/hashicorp/vault-plugin-database-couchbase v0.2.0
-	github.com/hashicorp/vault-plugin-database-elasticsearch v0.6.0
-	github.com/hashicorp/vault-plugin-database-mongodbatlas v0.2.0
+	github.com/hashicorp/vault-plugin-database-couchbase v0.1.1-0.20201022222321-52b89dc4ff04
+	github.com/hashicorp/vault-plugin-database-elasticsearch v0.5.6-0.20201022222154-56454997629c
+	github.com/hashicorp/vault-plugin-database-mongodbatlas v0.1.3-0.20201022222205-830fcbc5fdd2
```

Should these be tracking the release branches of each plugin repo? i.e. something more like
```suggestion
	github.com/hashicorp/vault-plugin-database-couchbase v0.2.1
	github.com/hashicorp/vault-plugin-database-elasticsearch v0.6.1
	github.com/hashicorp/vault-plugin-database-mongodbatlas v0.2.1
```

---

No because that's not what the original PR did. Those tags ended up being created afterwards. They're effectively the same, but I didn't want to hold up that PR to wait for tagging of the repos.

---

Ok, since this contains so many other changes, I'm ok with updating the tagging and versions after this.

---

This is fine, but you'd have to do a follow-up PR (against the release branch) after this is merged to use the actual release tags as Theron is suggesting.

---

I can update this to use the release tags now.